test/jobs/purge_job_test.rb

```diff
@@ -0,0 +1,81 @@
+# frozen_string_literal: true
+
+require 'test_case'
+
+class PurgeJobTest < ActiveSupport::TestCase
+  def setup
+    # Allowed to purge immediately for the tests
+    ADMIN_OPTIONS['job_run_email'] = 'test@test.com'
+    Patient.delete_all
+    Assessment.delete_all
+    ReportedCondition.delete_all
+    Symptom.delete_all
+  end
+
+  def teardown
+    ADMIN_OPTIONS['job_run_email'] = nil
+  end
+
+  test 'sends an email with all purged monitorees' do
+    patient = create(:patient, monitoring: false, purged: false)
+    patient.update(updated_at: (ADMIN_OPTIONS['purgeable_after'].minutes + 14.days).ago)
```

Comment:
Why is the `+ 14.days` needed for this?

Reply:
With the interaction between all of the admin options in the eligible for purge scope (such as making sure the reminder email went out after the monitoree hasn't been updated) and the way those defaults are set in the code i needed to make sure that the test wouldn't intermittently fail based on the day of the week so I set the updated at to a value far in the past.